packages/components/src/avatar/src/Avatar.css

```diff
@@ -65,13 +58,31 @@
     align-items: center;
 }
 
+.o-ui-avatar-group .o-ui-avatar:not(.o-ui-avatar-group-remainings) {
+    border: 1px solid var(--hop-neutral-surface);
+}
+
+.o-ui-avatar-group .o-ui-avatar-xs:not(:first-child):not(.o-ui-avatar-group-remainings),
+.o-ui-avatar-group .o-ui-avatar-sm:not(:first-child):not(.o-ui-avatar-group-remainings) {
+    margin-left: calc(var(--hop-space-inline-xs) * -1);
```

Comment:
I would not prevent a merge for this but I usually do -1 first as you can easily understand where the calc is going.